docs/application/dotnet/guides/nui/videoview.md

```diff
@@ -1,58 +1,99 @@
+---
+keyword: video, VideoView, playback, play, NUI, ResourceUrl, volume,
+---
+
 # VideoView
 
-The `VideoView` class is a control for video playback and display. It provides minimum functions for playback (play, pause, stop, forward, and backward). Some options, such as volume, can be controlled through the control properties.
+The [VideoView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.VideoView.html) class is a control for video playback and display. It provides minimum functions for playback (play, pause, stop, forward, and backward). Some options, such as volume, can be controlled through the control properties. NUI supports many kinds of video format, such as `.avi`, `.3gp` and `mp4`.
 
  > **Note**  
- > You must install a video plugin to enable `VideoView`.
+ > The `VideoView` control does not use any privileges APIs on its own. However, if you use video files in a specific device storage, the application requires privileges to access the storage.
 
-**Figure: VideoView**
 
-![VideoView](./media/dali_videoview.png)
+## Create a VideoView
 
- > **Note**  
- > The VideoView control does not use any privileges APIs on its own. However, if you use video files in a specific device storage, the application requires privileges to access the storage.
+The following basic example shows how to create a `VideoView` object:
 
-In this tutorial, the following subjects are covered:
+**Figure: Basic VideoView**
 
-[VideoView event](#1)<br>
-[Creating a VideoView](#2)<br>
-[VideoView Properties](#3)<br>
+![Basic VideoView](./media/basicVideoview.png)
 
-<a name="1"></a>
-## VideoView event
+1.  To use the `VideoView` class, add the following namespaces:
 
-The following table lists the basic signal provided by the `VideoView` class.
+    ```cs
+    using Tizen.NUI;
+    using Tizen.NUI.BaseComponents;
+    ```	    
+  
+2.   The video file is assumed to be in the resources directory. Create an instance of the `VideoView` class and use the `ResourceUrl` property to pass the path to the video file. Set `WidthResizePolicy` to make `VideoView` instance use full width of the window and set `HeightResizePolicy` to maintain aspect ratio of video. In the end invoke `Play()` method to start video:
```

Comment:
In the end invoke `Play()` method to start video:

It could be a bit confusing to see end & start in the same line, hence recommend to change to:

And then you can invoke the `Play()` method to start video: